File: Thought-works/Round-1-HackerRank/removeLetters.py
# remove letter from given string 

"""
i/p: "hello",["e","o"]
o/p: "hll"
"""

def remove(word,letters):
    b = []
    for i in word:
        if i in letters:
            continue
        b.append(i)
    return "".join(b)
res = remove("hello",["e","o"])
print(res)

File: Thought-works/Round-1-HackerRank/Removeletters.java
public class Remove {

    // i/p: "hello",["e","o"]
    // o/p: "hll"

    static void print(String s, char[] a) {

        
        for(int i=0;i<s.length();i++){
            int flag = 0;
            for(int j =0; j< a.length;j++){
                if(s.charAt(i) == a[j]){
                    flag = 1;
                    break;
                }
            }
            if(flag == 0){
                System.out.print(s.charAt(i));
            }
        }
    }

    public static void main(String[] args) {
        char[] arr = new char[2];
        arr[0] = 'e';
        arr[1] = 'o';
        
        print("hello",arr);

    }

}
